src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -118,433 +131,633 @@ def test_build_flowsheet(self, model):
+            0
+        ].value == pytest.approx(440.422915, 1e-4)
+        assert scaled_model.fs.solex_rougher_scrub.mscontactor.aqueous_outlet.flow_vol[
+            0
+        ].value == pytest.approx(0.09, 1e-4)
+        assert scaled_model.fs.solex_rougher_strip.mscontactor.aqueous_outlet.flow_vol[
+            0
+        ].value == pytest.approx(0.09, 1e-4)
+        assert scaled_model.fs.acid_feed1.outlet.flow_vol[0].value == pytest.approx(
+            0.09, 1e-4
+        )
+        assert scaled_model.fs.acid_feed2.outlet.flow_vol[0].value == pytest.approx(
+            0.09, 1e-4
+        )
+        assert scaled_model.fs.acid_feed3.outlet.flow_vol[0].value == pytest.approx(
+            9, 1e-4
+        )
+        assert scaled_model.fs.rougher_sep.inlet.flow_vol[0].value == pytest.approx(
+            62.01, 1e-4
+        )
+        assert scaled_model.fs.load_sep.inlet.flow_vol[0].value == pytest.approx(
+            440.422915, 1e-4
+        )
+        assert scaled_model.fs.scrub_sep.inlet.flow_vol[0].value == pytest.approx(
+            0.09, 1e-4
+        )
+        assert scaled_model.fs.rougher_mixer.outlet.flow_vol[0].value == pytest.approx(
+            62.01, 1e-4
+        )
+        assert scaled_model.fs.sc_circuit_purge.inlet.flow_vol[
+            0
+        ].value == pytest.approx(6.201, 1e-4)
+        assert scaled_model.fs.solex_cleaner_load.mscontactor.aqueous_outlet.flow_vol[
+            0
+        ].value == pytest.approx(5.76, 1e-4)
+        assert scaled_model.fs.solex_cleaner_strip.mscontactor.aqueous_outlet.flow_vol[
+            0
+        ].value == pytest.approx(0.09, 1e-4)
+        assert scaled_model.fs.cleaner_org_make_up.outlet.flow_vol[
+            0
+        ].value == pytest.approx(6.201, 1e-4)
+        assert scaled_model.fs.cleaner_mixer.outlet.flow_vol[0].value == pytest.approx(
+            62.01, 1e-4
+        )
+        assert scaled_model.fs.cleaner_sep.inlet.flow_vol[0].value == pytest.approx(
+            62.01, 1e-4
+        )
+        assert scaled_model.fs.leach_sx_mixer.outlet.flow_vol[0].value == pytest.approx(
+            440.422915, 1e-4
+        )
+        assert scaled_model.fs.cleaner_purge.inlet.flow_vol[0].value == pytest.approx(
+            6.201, 1e-4
+        )
+        assert scaled_model.fs.sl_sep1.recovered_liquid_outlet.flow_vol[
+            0
+        ].value == pytest.approx(434.662915, 1e-4)
+        assert scaled_model.fs.sl_sep2.recovered_liquid_outlet.flow_vol[
+            0
+        ].value == pytest.approx(0.063, 1e-4)
+        assert scaled_model.fs.precip_sep.inlet.flow_vol[0].value == pytest.approx(
+            0.063, 1e-4
+        )
+        assert scaled_model.fs.precip_sx_mixer.outlet.flow_vol[
+            0
+        ].value == pytest.approx(5.76, 1e-4)
+        assert scaled_model.fs.precip_purge.inlet.flow_vol[0].value == pytest.approx(
+            0.0063, 1e-4
+        )
+
+    def test_main(self):
+        main()
```

Comment:
Does `main` have any meaningful assertions in it?

Reply:
No - I had originally included it here for code coverage reasons, but I've since cleaned up the main function, so I don't believe this is necessary anymore (can't check the code coverage now due to the seemingly transient  testfailure)